docs/src/user_docs/assembly/cryptographic_operations.md

```diff
@@ -6,6 +6,7 @@ Miden assembly provides a set of instructions for performing common cryptographi
 
 | Instruction    | Stack_input     | Stack_output   | Notes                                      |
 | -------------- | --------------- | -------------- | ------------------------------------------ |
+| hash <br> - *(1 cycle)*  | [A, ...] | [B, ...] | $\{B\} \leftarrow hash(A)$ <BR> where, $hash()$ computes a 1-to-1 Rescue Prime Optimized hash. |
```

Comment:
It appears that the VM cycle count for this operation is incorrect. Since this isn't a core VM instruction, the cycle count should be greater than 1. After calculating, I think the cycle count should be 19 instead.

Reply:
good catch.  I calculated 19 as well and have updated accordingly.  Thank you for your review on this!